Makefile

```diff
@@ -7,7 +7,17 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	targets/openshift/images.mk \
 )
 
-IMAGE_REGISTRY :=registry.ci.openshift.org
+IMAGE_REGISTRY := registry.ci.openshift.org
+# Use podman to build the must gather image
+# Please ensure you have podman installed on your machine for this
+IMAGE_BUILD_BUILDER := podman build
```

Comment:
Given imagebuilder has been used until now without ability to change it all the scripts calling the makefile will most likely expect imagebuilder as well. I still prefer to keep the default as it is.

Reply:
Fixed to use a BUILDER env to specify podman as a builder while leaving imagebuilder as the default